Assets/Python/GreatPeople.py

```diff
@@ -2042,11 +2045,20 @@ def setup():
 		"Pieter Stuyvesant", # 17th 
 		"Jan van Riebeeck", # 17th 
 		"Jan Coen", # 17th 
+		iIndustrial,
+		"Brenninkmeijer", # 19th 
```

Comment:
I prefer full names, so please choose one as a representative GP, like I did with the Wright brothers.